src/github/nag.rs

```diff
@@ -395,7 +395,7 @@ fn evaluate_pendings() -> DashResult<()> {
                         proposal.id, why));
         }
 
-        let majority_complete = num_outstanding_reviews < num_complete_reviews;
+        let majority_complete = num_outstanding_reviews <= 2 * num_complete_reviews;
```

Comment:
This seems wrong: with 2 complete reviews and 4 outstanding reviews, this evaluates to true. ^^